content/actions/learn-github-actions/contexts.md

```diff
@@ -57,6 +57,8 @@ In order to use property dereference syntax, the property name must:
 - start with `a-Z` or `_`.
 - be followed by `a-Z` `0-9` `-` or `_`.
 
+Attempting to dereference a non-existent property will evaluate to an empty string.
```

Comment:
To bring this into line with our style guide, I'm going to suggest a rephrasing too.

```suggestion
If you attempt to dereference a non-existent property, it will evaluate to an empty string.
```